ci-operator/step-registry/openstack/provision/sriov-networknodepolicy/openstack-provision-sriov-networknodepolicy-commands.sh

```diff
@@ -54,12 +56,20 @@ then
 	source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
-create_sriov_networknodepolicy "sriov1" "${OPENSTACK_SRIOV_NETWORK}" "vfio-pci"
+if [[ "${OPENSTACK_SRIOV_NETWORK}" == "mellanox-sriov" ]]; then
+    SRIOV_DEVICE_TYPE="netdevice"
+    IS_RDMA="true"
+else
+    SRIOV_DEVICE_TYPE="vfio-pci"
+    IS_RDMA="false"
+fi
+
+create_sriov_networknodepolicy "sriov1" "${OPENSTACK_SRIOV_NETWORK}" "${SRIOV_DEVICE_TYPE}" "${IS_RDMA}"
 
 if [[ "${OPENSTACK_DPDK_NETWORK}" != "" ]]; then
     if oc get MachineConfig/99-vhostuser-bind >/dev/null 2>&1; then
         echo "vhostuser is already bound to the ${OPENSTACK_DPDK_NETWORK} network."
```

Comment:
Just to confirm, this is harmless right?
I saw that the sriov node policy reported this.

Reply:
yeah, if you saw this in the logs it means that it's a job that deploys the `vhostuser` step registry and will create a network policy for DPDK.